hack/config

```diff
@@ -13,6 +13,7 @@ LIVENESS_PROBE_VERSION="v2.3.0"
 CSI_SNAPSHOT_VERSION="v4.2.1"
 CSI_SIG_STORAGE_PROVISIONER_VERSION="v2.2.1"
 KUBE_RBAC_PROXY_VERSION="4.10"
+KUBEVIRT_PLUGIN_VERSION="latest"
```

Comment:
maybe we should add a "UI" or "CONSOLE" to the name, as it is it can get confused with the main kubevirt operators,
for example ''KUBEVIRT_CONSOLE_PLUGIN_VERSION", does that make sense to you ?